test/unit/app/controllers/permissions/mocks.js

```diff
@@ -180,11 +189,15 @@ const CAVEATS = {
    * @returns {Object} An eth_accounts exposedAccounts caveats
    */
   eth_accounts: (accounts) => {
-    return {
+    return [{
+      type: 'limitResponseLength',
```

Comment:
We should probably have a `CAVEAT_TYPES` enum to go with the `CAVEAT_NAMES`.

Reply:
Agreed

Reply:
Done

Reply:
We should use it too 😛

Reply:
nit: We should use it for all instances of caveat types in the `eth_accounts` mock (i.e., right here), since we already use it for names.

Reply:
Sorry about that @rekmarks -- got it taken care of now.